tasks/keepalived_install.yml

```diff
@@ -59,6 +59,19 @@
   when:
     - ansible_os_family | lower == 'debian'
     - check_if_present | changed
+    - prevent_start_mask_systemd is not defined
```

Comment:
I'd rather not define another variable, and use ansible_service_mgr instead.
so I'd adapt the condition here by:
```
when:
  - ansible_os_family | lower == 'debian'
  - check_if_present | changed
  - ansible_service_mgr is defined
  - ansible_service_mgr != 'systemd'
```

Reply:
Done